.env.example

```
@@ -25,6 +25,9 @@ OIDC_AUTHORITY=
 OIDC_CLIENT_ID=
 OIDC_USERNAME_PROPERTY=
 OIDC_CERTS_URL=
+OIDC_TOKEN_ENDPOINT=
+USE_OIDC_ROLES=false
```

```suggestion
USE_OIDC_ROLES=
```

This is probably a limitation from the `decouple` lib but passing `false` here is being interpreted as a string and then the check `if USE_OIDC_ROLES:` is true, even when we want to have it false. So better to leave it empty if we want this to be false.